_events/discussions/event-013.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Research Software Directories: What, Why and How?"
+authors:
+    - &speaker
+      name: Mark Woodbridge
+      orcid: 0000-0002-0650-9751
+      is_speaker: true
+      affiliation: 1
+    - &speaker
+      name: Vanessa Sochat
+      orcid: 0000-0002-4387-3819
+      is_speaker: true
+      affiliation: 3
+    - &speaker
+      name: Jurriaan Spaaks
```

Comment:
```suggestion
    - name: Jurriaan Spaaks
```
see above.

Reply:
Will take care tomorrow, thanks for spotting.

Reply:
thanks a lot @eileen-kuehn! Thinking about it, I'd propose we make your life easier and just remove the `author: *speaker` and don't show the author profile at all in the upper left corner. The only advantage of keeping the _author_ is that the profile in the upper left corner can show other things, like website, github, etc.

Recap: The `author` is show in the sidebar 
![image](https://user-images.githubusercontent.com/9960249/90592529-13415d80-e1e6-11ea-990f-72de845459c1.png)

and the `authors` (plural) are shown below the title
![image](https://user-images.githubusercontent.com/9960249/90592556-26542d80-e1e6-11ea-96ed-4f8f5d936773.png)

Reply:
If you like, I can fix this within this PR and make a PR to sorse-data-filter to remove the `author` field

Reply:
I am currently sitting in a train, so happy to take care about it :) However, the `speaker` thing is easy to solve. So feel free to adapt the visualisation but for the moment I tend to leave the data as is to give us more flexibility.

Reply:
Alright :+1: I'll disable the author profile in the sidebar later when I am in front of the computer and we leave the `author` in the front matter :smiley: